src/sentry/static/sentry/app/views/projectDetail/projectDetail.tsx

```diff
@@ -109,6 +120,12 @@ class ProjectDetail extends AsyncView<Props, State> {
             <Layout.Body>
               <Layout.Main>
                 <ProjectScoreCards />
+                <ProjectCharts
+                  api={this.chartsApi}
```

Comment:
Should `ProjectCharts` be wrapped in `withApi()` instead?

Reply:
😑 yep, my bad